src/client.ts

```diff
@@ -513,6 +552,17 @@ export class Connection {
   private getBufferSizeParams(): BufferSizeParams {
     return { maxSize: this.frameMax, ...this.bufferSizeSettings }
   }
+
+  private async initNewClient(streamName: string, leader: boolean): Promise<Client> {
+    const [metadata] = await this.queryMetadata({ streams: [streamName] })
+    const chosenNode = leader ? metadata.leader : sample([metadata.leader, ...(metadata.replicas ?? [])])
+    if (!chosenNode) {
+      // need some help with this error message -- what might be better than this? --tarza (20/12/23)
+      throw new Error(`Stream was not found on any node`)
```

Comment:
I think a `Stream not found` message is ok

... so could we remove the comment and the `tarza` entry in the cspell configuration file?

Reply:
Done!